src/serverMain/kotlin/com/studo/campusqr/endpoints/ReportData.kt

```diff
@@ -46,15 +66,17 @@ suspend fun AuthenticatedApplicationCall.returnReportData() {
 
   val impactedUsersEmailsTask: Deferred<List<String>> = serverScope.async(Dispatchers.IO) {
     runOnDb {
-      // Keep logic in sync with listAllActiveCheckIns()
-      val previousInfectionHours: Int = getConfig("previousInfectionHours")
-      val nextInfectionHours: Int = getConfig("nextInfectionHours")
+      val transitThresholdSeconds: Int = getConfig("transitThresholdSeconds")
 
       // We need to probably optimize performance here in the future
-      val otherCheckIns = reportedUserCheckIns.flatMap { checkIn ->
+      val otherCheckIns = reportedUserCheckIns.flatMap { reportedUserCheckin ->
         getCollection<CheckIn>().find(
-            CheckIn::locationId equal checkIn.locationId,
-            CheckIn::date.inRange(checkIn.date.addHours(-previousInfectionHours), checkIn.date.addHours(nextInfectionHours))
+            CheckIn::locationId equal reportedUserCheckin.locationId,
+            CheckIn::date lowerEquals (reportedUserCheckin.checkOutDate ?: Date()).addSeconds(transitThresholdSeconds),
```

Comment:
I think we should only use one `now` date in this function, especially since this function might evaluate for some seconds @functionaldude